example/myapp/bin/myapp.dart

```diff
@@ -1,36 +1,75 @@
+import 'dart:math';
+import 'dart:typed_data';
+
 import 'package:realm_dart/realm.dart';
 
 part 'myapp.g.dart';
-/*
+
+@RealmModel()
 class _Car {
-  @RealmProperty()
   late String make;
+  late _Person? person;
```

Comment:
I have removed the entire example app in this PR. We need to create a new. The one in this PR could hardly be called an example, and the one in master is too stale)

Reply:
My suggestion was to remove only the changes to the example. Lets leave the package structure as it is since we need to update the examples and have it there. Also the same for the flutter example. These apps just need an update to the API changes we introduced between the preview and the current version

Reply:
My thinking was that since the code is mostly out-commented on master anyway, it would easier to just create a new example. Instead, I will drop the commits that removes the example (both this and the flutter one), undo the changes done on this branch and do minimal changes to make the original example work with the new generator.